diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index 5a4c24f50b4852eeff2e606f9b9f6fc77f99d89c..de823e7a23fae375400ab26de4a0a0f3242437d2 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -33,7 +33,7 @@ exports.networks = [
     { chainId: 63n, shortName: 'metc' },
     { chainId: 69n, shortName: 'okov' },
     { chainId: 71n, shortName: 'cfxtest' },
-    { chainId: 81n, shortName: 'joc' },
+    { chainId: 81n, shortName: 'sby' },
     { chainId: 82n, shortName: 'meter' },
     { chainId: 83n, shortName: 'meter-test' },
     { chainId: 88n, shortName: 'vic' },
